airflow.yaml

```diff
@@ -63,6 +66,14 @@ pipeline:
       python -m build --wheel
       pip install dist/*.whl
 
+  - runs: |
+      # CVE-2024-6345 GHSA-cx63-2mw6-8hw5
+      # setuptools comes from airflow/providers/google/provider.yaml having
+      # gcloud-aio-auth>=4.0.0,<5.0.0 . gcloud-aio-auth 4 is backlevel and has
+      # setuptools in it's pyproject.toml 'tool.poetry.dependencies'
+      # The tldr; For that case it is not needed in runtime.
+      pip uninstall --yes setuptools
```

Comment:
bah. i wasn't in the venv when i did this :-(
taking a quick look at separating build and runtime venv. if that doesn't work, then the change here is just:

```
./venv/bin/pip uninstall --yes setuptools
```